webview/src/experiments/components/App.test.tsx

```diff
@@ -344,56 +346,10 @@ describe('App', () => {
         })
       }
 
-      const [workspace, experimentRunningInWorkspace] = screen.getAllByText(
-        EXPERIMENT_WORKSPACE_ID
-      )
-
-      testClick(workspace, EXPERIMENT_WORKSPACE_ID)
-      testClick(experimentRunningInWorkspace, 'exp-83425')
-      testClick(screen.getAllByText('main')[1], 'main')
-      testClick(screen.getByText('[exp-e7a67]'), 'exp-e7a67')
-    })
-
-    it('should send a message to the extension to toggle an experiment when Enter or Space is pressed on the row', () => {
```

Comment:
[F] I spent 20-30 minutes trying to fix this test but could not come up with anything that worked. The behaviour does work.

Reply:
Maybe I'm misunderstanding this feature, but when I tabbed onto a table row and pressed enter/space nothing was toggled 🤔

Reply:
Focus needs to be on the plot icon. I'll record a demo later

Reply:
https://github.com/iterative/vscode-dvc/assets/37993418/623dd675-0d13-41f1-8eb9-eaff1841f5b1